app/src/main/java/com/amaze/filemanager/filesystem/FileUtil.java

```diff
@@ -619,6 +620,35 @@ public static boolean mkfile(final File file, Context context) {
     return false;
   }
 
+  public static boolean mktextfile(String data, String path, String fileName) {
+
+    File f = new File(path + "/" + fileName + ".txt");
+    FileOutputStream out = null;
+    OutputStreamWriter outputWriter = null;
+    if (f.exists()) {
+      f.delete();
+    }
+    try {
+      f.createNewFile();
+      out = new FileOutputStream(f);
```

Comment:
Pass a second argument as false to FileOutputStream constructor, it'll overwrite instead of appending. No need to delete the file first.